x-pack/plugins/security/server/authentication/unauthenticated_page.test.tsx

```diff
@@ -28,6 +28,25 @@ describe('UnauthenticatedPage', () => {
         originalURL="/some/url?some-query=some-value#some-hash"
         buildNumber={100500}
         basePath={mockCoreSetup.http.basePath}
+        customBranding={{}}
+      />
+    );
+
+    expect(body).toMatchSnapshot();
+  });
+
+  it('renders as expected with custom title', async () => {
```

Comment:
__nit:__ based on the answer to my question in `prompt_page.tsx` (e.g. no need to change `Elastic` title) you might need to change this test to "with custom favicon" or something.